### DISCUSSION TITLE
How to change the default chevron icon in Accordion to +/- ?

### DISCUSSION THREAD

**Discussion by sivaranjansahu** [chakra-ui/chakra-ui#4706] [Q&A] [answered]
I am trying to figure out how to change the chevron icon in the Accordion items to something else. I think I need to do that in the theme, but not sure how to do. Can anyone help , please ? thanks

**Comment by payapula** [comment, marked as answer]
Hi @sivaranjansahu 

They have an example for this exact use case in the docs: [Accessing the internal state - Accordion | Chakra Docs](https://chakra-ui.com/docs/disclosure/accordion#accessing-the-internal-state)

And here is the live example of how to achieve what you are looking for - https://codesandbox.io/s/thirsty-cartwright-v6mqo?file=/src/index.js

let me know if this helps. :)

**Reply by sivaranjansahu** [reply]
Thank you so much!!

**Reply by FredPerr** [reply]
The problem seems to be deeper than that because the animation is no longer usable. The `AccordionIcon` should extend the `as` property correctly in TypeScript. It works fine if ignoring type error while doing the following:
`
<AccordionIcon as={HiChevronDown}/>
`

_`HiChevronDown`_ is just an icon from `react-icons`.

**Reply by ahmedshantodev** [reply]
@FredPerr  thanks 🖤
